src/widgets/Nav/Panel.tsx

```diff
@@ -112,8 +113,8 @@ const Panel: React.FC<Props> = ({
         <CloseIcon />
       </MobileOnlyButton>
       <LinkBlock>
-        {config.map((entry) => {
-          if (entry.items) {
+        {links.map((entry) => {
+          if ("items" in entry) {
```

Comment:
TIL: You need the Typescript type guard in order for this to work.
https://www.typescriptlang.org/docs/handbook/advanced-types.html